extras/rya.streams/client/src/main/java/org/apache/rya/streams/client/command/StreamResultsCommand.java

```diff
@@ -143,6 +148,8 @@ public void execute(final String[] args) throws ArgumentsException, ExecutionExc
             sparql = sQuery.get().getSparql();
         } catch (final Exception e) {
             throw new ExecutionException("Problem encountered while closing the QueryRepository.", e);
+        } finally {
+            queryRepo.stop();
```

Comment:
I'm not sure that any of these stops are evaluated since the repository is never started.